Core/FieldHandler/EzCountry.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+namespace Kaliop\eZMigrationBundle\Core\FieldHandler;
+
+use eZ\Publish\API\Repository\ContentTypeService;
+use eZ\Publish\Core\FieldType\Country\Value as CountryValue;
+use eZ\Publish\Core\FieldType\Selection\Value as SelectionValue;
+use Kaliop\eZMigrationBundle\API\FieldValueImporterInterface;
+
+class EzCountry extends AbstractFieldHandler implements FieldValueImporterInterface
+{
+    private $contentTypeService;
+
+    private $countries;
+
+    public function __construct(ContentTypeService $contentTypeService, array $countries)
+    {
+        $this->contentTypeService = $contentTypeService;
+        $this->countries = $countries;
+    }
+
+    /**
+     * Creates a value object to use as the field value when setting a country field type.
+     *
+     * @param $fieldValue string|string[] should work
+     * @param array $context The context for execution of the current migrations. Contains f.e. the path to the migration
+     * @return CountryValue
+     */
+    public function hashToFieldValue($fieldValue, array $context = array())
+    {
+        if ($fieldValue === null) {
+            return new CountryValue();
+        }
+
+        $fieldSettings = $this->loadFieldSettings($context['contentTypeIdentifier'], $context['fieldIdentifier']);
+
+        $countries = [];
+        if (!$fieldSettings['isMultiple'] && is_string($fieldValue)) {
```

Comment:
what happens if field is not multiple but user sends in an array of values?

Reply:
Core says you cant pass an array in that case.